samples/go/csv/read_test.go

```diff
@@ -133,6 +133,28 @@ g,h,i,j
 `
 	testTrailingHelper(t, dataString)
 }
+func TestEscapeStructFieldFromCSV(t *testing.T) {
```

Comment:
Are you running `gofmt` in your editor? Should be a blank line between functions.